src/MicrosoftTeams.ts

```diff
@@ -588,6 +588,31 @@ namespace microsoftTeams {
     };
   }
 
+  /**
+   * enable print capability
+   */
+  export function enablePrintCapability(): void {
+    ensureInitialized();
+    /**
+     * adding ctrl+P and cmd+P handler
+     */
+    document.addEventListener("keydown", function(event: KeyboardEvent): void {
```

Comment:
If a developer makes the mistake of calling `enablePrintCapability` twice would we register two of these handlers? Might be a good idea to somehow protect against that.